### ISSUE TITLE
[bug] Moving a file/dir into another directory with a file/dir the same name gives missleading error

### ISSUE THREAD

**Issue by raguay** [fman-community/fman#348]
Moving a file/dir into another directory with a file/dir the same name gives a miss-leading error. It says you can not move the file onto itself, but it is a different file, different sizes, and different locations. They just happen to have the same name. Copy doesn't seem to cause the same issue because I do it all the time. Just the moving a file does.

**Comment by mherrmann** [comment]
Sorry, how do you mean? I created the following directory structure:

 * `src/`
   * `b/`
     * `file.txt`
 * `dst/`
   * `b/`

I then opened `src` in one pane and `dst` in the other. Then, I used <kbd>F6</kbd> to move `b` from `src` to `dst`. I did not get an error. As I had expected, what happened was that `src/b` disappeared and `file.txt` ended up at `dst/b/file.txt`.

**Comment by raguay** [comment]
Yea, I wrote that very poorly and now I can't seem to recreate it. I've gone back over the steps and it seems okay. I was moving programs in and out of /Applications trying to fix the programs registration with the OS (moving out of /Applications using fman and moving them back using Finder to get the doorkeeper from saying they are in a read-only file space. What a pain!).

The programs moving crashed and I had to restart it with part of the files moved and part not moved. That is when I got the message that it was moving the directory/files onto itself, but it wasn't. I bet it was a funny think interacting with the macOS doorkeeper. Not sure.
